docs/running-calyx/interfacing.md

```diff
@@ -0,0 +1,64 @@
+# Interfacing with Calyx Programs
+
+In order to run RTL designs created from Calyx programs, toplevel `reset` and `go`
+signals must be interfaced with correctly.
+
+Namely:
+1. The `reset` signal must be asserted then deasserted, to initialize the state inside
+control registers correctly.
+2. The `go` signal must be asserted for as long as the module is running. Deasserting
+the `go` signal before a component's `done` signal is asserted will lead to
+[undefined behavior][go-done].
+
+Asserting the `reset` and `go` signals in this order is important. Otherwise the toplevel
+component will begin running with garbage data inside of control registers.
+
+
+Interfacing with RTL designs in this way becomes relevant when writing harnesses/testbenches
+to execute programs created with Calyx.
```

Comment:
Maybe this should move to the top of the document, to help explain who should care about the rest of the document?